bson/writer.go

```diff
@@ -50,32 +50,11 @@ type ValueWriter interface {
 	WriteUndefined() error
 }
 
-// ValueWriterFlusher is a superset of ValueWriter that exposes functionality to flush to the underlying buffer.
-//
-// Deprecated: ValueWriterFlusher will not be supported in Go Driver 2.0.
-type ValueWriterFlusher interface {
```

Comment:
Only used by `ValueWriterPool.GetAtModeElement()`, which is only called by `MarshalValueWithRegistry()`.